data/hai/hai reveal 1 intro.txt

```diff
@@ -695,12 +713,18 @@ mission "Hai Leaks Response 3"
 			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
 			`	You get down to business when Sayari inquires how your journey went.`
 			choice
+				`	Keep silent and let the meeting begin.`
```

Comment:
It's strange to me that saying nothing skips dialogue, and it's a bit rude that the player can ignore Sayari's question. 

I only noticed now that Sayari also gives important information here about the pirates here, and it's weird that you never get the opportunity to tell this to Teneep before the stuff with Scar's Legion, because otherwise the player has to go to fight pirates to get Teneep to know what the player knew already.

Reply:
Teeneep already knows about the pirate activity in Hai space. (It is common knowledge.) She doesn't know about Scar's Legion specifically. That mission sequence sets up her strong reaction to the Elder's offer to ship Hai weapons to the humans.

Reply:
I meant the "rewards" the Elders gave pirates for keeping the secret contained. The strong reaction is good, but I didn't know why the player couldn't have just pointed this information out earlier, something like "Sayari said that the Elders have some arrangement with the pirates. Do you know what this means?"

Reply:
I meant to take out the "You get down to business..." line. It is gone now, and the changes after it make more sense now.

Reply:
I want to limit this PR to dialog cleanup only, and your suggested dialog connection (telling Teeneep what Sayari said) is a story change, so it belongs in a different PR.

Reply:
Okay, that's good.